css/style.css

```diff
@@ -0,0 +1,3154 @@
+  margin-top: -1px;
+}
+
+.update-inside__iconThird {
+  width: 31.07px;
+  height: 10.4px;
+  padding-left: 4px;
+  margin-top: -4px;
+}
+
+.update-inside__iconFourth {
+  width: 44px;
+  height: 13.12px;
+  animation-duration: 1s;
+  animation-name: blickOne;
+  animation-iteration-count: infinite;
+  animation-direction: alternate;
+  transform-origin: 100% 0%;
+  margin-left: -2px;
+}
+
+@keyframes blickOne {
+  0% {
+    opacity: 0;
+  }
+  
+  100% {
+    opacity: 1;
+  }
+}
+
+.update-inside__title {
+  padding-left: 15px;
+  display: flex;
+  flex-direction: column;
+}
+
+.update-inside__titleText {
+  margin: 0px;
+  font-size: 20px;
+  font-family: "MontserratSemiBold";
+  padding-bottom: 5px;
+}
+
+.update-inside__Text {
+  font-size: 12px;
+  font-family: "MontserratRegular";
+  margin: 0px;
+  width: 60%;
+}
+
+.description-inside__volume {
+  padding-top: 25px;
+  display: flex;
+  flex-direction: row;
+}
+
+.volume-inside__icon {
+  display: flex;
+  flex-direction: column;
+}
+
+.volume-inside__iconSmall {
+  width: 18px;
+  height: 12px;
+  padding-left: 13px;
+}
+
+.volume-inside__iconBig {
+  width: 36px;
+  height: 44px;
+  padding-left: 4.2px;
+  margin-top: -3px;
+}
+
+.volume-inside__description {
+  display: flex;
+  flex-direction: column;
+  padding-left: 16px;
+}
+
+.description-inside__title {
+  display: flex;
+  flex-direction: row;
+}
+
+.description-inside__titleNumb {
+  font-size: 35px;
+  font-family: "MontserratSemiBold";
+  margin: 0px;
+  margin-top: -10px;
+}
+
+.description-inside__titleText {
+  font-size: 20px;
+  font-family: "MontserratSemiBold";
+  padding-left: 10px;
+  margin: 0px;
+  align-self: flex-end;
+  padding-bottom: 4px;
+}
+
+.description-inside__desc {
+  font-size: 12px;
+  font-family: "MontserratRegular";
+  margin: 0px;
+}
+
+.description-inside__information {
+  display: flex;
+  flex-direction: row;
+  align-items: flex-end;
+  padding-top: 30px;
+}
+
+.information-inside__icon {
+  padding-bottom: 3px;
+  cursor: pointer;
+}
+
+.information-inside__iconEllipse {
+  position: relative;
+  width: 22px;
+  height: 22px;
+  z-index: 4;
+}
+
+.information-inside__iconI {
+  height: 16px;
+  width: 4px;
+  position: absolute;
+  z-index: 5;
+  margin-left: -13px;
+  padding-top: 2px;
+}
+
+.information-inside__title {
+  font-family: "MontserratBold";
+  padding-left: 5px;
+  font-size: 13px;
+  margin: 0px;
+  padding-bottom: 10px;
+  padding-right: 22px;
+}
+
+.description-inside__button {
+  margin: 0px;
+  color: white;
+  display: flex;
+  align-self: center;
+  font-family: "MontserratBold";
+  font-size: 13px;
+  border: 2px solid white;
+  box-shadow: none;
+  transition: box-shadow 0.6s linear;
+  padding: 10px 40px;
+}
+
+.description-inside__button:hover {
+  box-shadow: 0px 0px 0px 2px white;
+}
+
+.security {
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  position: relative;
+}
+
+.security__description {
+  background: #E51937;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Expected "#E51937" to be "#e51937" (color-hex-case)](https://app.codacy.com/gh/WebHeroSchool/tesla_project/pullRequest?prid=5520131)